infra/terraform/modules/user_nfs_softnas/main.tf

```diff
@@ -14,6 +14,10 @@ variable "num_instances" {
   default = 2
 }
 
+variable "softnas_ami_id" {
```

Comment:
I know it doesn't matter from Terraform (TF) perspective but I've been trying to structure TF modules to have input variables in the `inputs.tf` and output variables in the `outputs.tf`. 

I like that because then when I have a look at a module I didn't personally wrote I know that by looking at the `inputs.tf` I can have an idea of what input that module is expecting, etc...

Reply:
Sounds good. I've renamed this file inputs.tf.